ci/collect_logs.yml

```diff
@@ -0,0 +1,38 @@
+---
+- hosts: all
+  name: "Create zuul-output log dir"
+  gather_facts: false
+  tasks:
+    - name: "Create log dir"
+      ansible.builtin.file:
+        path: "{{ ansible_user_dir }}/zuul-output/logs"
+        state: directory
+        mode: "0755"
+
+- hosts: localhost
+  name: Collect logs on the controller
+  gather_facts: false
+  tasks:
+    - name: Create log dir
+      ansible.builtin.file:
+        path: "{{ ansible_user_dir }}/zuul-output/logs/controller"
+        state: directory
+        mode: "0755"
+
+    - name: Copy ansible logs
+      ansible.builtin.shell: |
+        cp {{ ansible_env.HOME }}/test_run_result.out .
```

Comment:
test_run_result.out actually isn't in $HOME. It's in playbook_dir by default.

I think you should be able to just reference {{ playbook_dir }} since all the ci/ playbooks are in teh same place.